src/sbt-test/sbtbiopet/developtool/test

```diff
@@ -0,0 +1,17 @@
+# Test assembly
+> assembly
+$ exists target/scala-2.11/DummyTool-assembly-0.1-SNAPSHOT.jar
+
+> publishLocal
```

Comment:
I you mean `package` instaed of `publishLocal`. Publish local is package + copy to ~/.ivf2/local